src/EditorFeatures/Core.Wpf/IWpfTextViewExtensions.cs

```diff
@@ -19,6 +19,12 @@ void firstLayout(object sender, TextViewLayoutChangedEventArgs args)
             {
                 view.VisualElement.Dispatcher.BeginInvoke(new Action(() =>
                 {
+                    var newHeight = view.LineHeight * view.TextBuffer.CurrentSnapshot.LineCount;
```

Comment:
can you add a comment why this is necessary?  For example, i can see: ```view.VisualElement.Height = view.LineHeight * view.TextBuffer.CurrentSnapshot.LineCount;``` a few lines above this.  So why would would this have any effect?  Thanks!

Reply:
Because they are executed at different points in time and may not give the same result since they rely on `view.LineHeight` which is mutable. They may give the same value sometimes, but by coincidence, and that's why this bug is not obvious and hasn't been fixed earlier.

I've removed the first `view.VisualElement.Height = ...` with the last commit since it's not necessary in VS2017.